flopy/modflow/mf.py

```diff
@@ -293,16 +299,21 @@ def modelgrid(self):
             ibound = None
 
         if self.get_package("disu") is not None:
-            self._modelgrid = Grid(
-                grid_type="USG-Unstructured",
-                top=self.disu.top,
-                botm=self.disu.bot,
-                idomain=ibound,
-                proj4=self._modelgrid.proj4,
-                epsg=self._modelgrid.epsg,
-                xoff=self._modelgrid.xoffset,
-                yoff=self._modelgrid.yoffset,
-                angrot=self._modelgrid.angrot,
+            # self._modelgrid = Grid(
+            #    grid_type="USG-Unstructured",
+            #    top=self.disu.top,
+            #    botm=self.disu.bot,
+            #    idomain=ibound,
+            #    proj4=self._modelgrid.proj4,
+            #    epsg=self._modelgrid.epsg,
+            #    xoff=self._modelgrid.xoffset,
+            #    yoff=self._modelgrid.yoffset,
+            #    angrot=self._modelgrid.angrot,
+            # )
```

Comment:
Should probably delete these commented lines.  Also, is there a reason we are not retaining the proj4, epsg, xoff, yoff, and angrot arguments?